src/index.ts

```diff
@@ -500,3 +500,44 @@ export function monzoToCents(monzo: Monzo) {
   }
   return Math.log1p(Number(delta) / Number(denominator)) * NATS_TO_CENTS;
 }
+
+/**
+ * Given fraction p/q calculate log(abs(p*q)).
+ * @param value Rational number or an array of its prime exponents.
+ * @returns The Tenney-height of the number.
+ */
+export function tenneyHeight(value: Monzo | FractionValue) {
+  if (Array.isArray(value)) {
+    return dotPrecise(
+      value.map(x => Math.abs(x)),
+      LOG_PRIMES
+    );
+  }
+  const {n, d} = new Fraction(value);
+  return Math.log(n) + Math.log(d);
+}
+
+/**
+ * Given fraction p/q calculate sopfr(p) + sopfr(q), ignoring sign.
+ * @param value Rational number or an array of its prime exponents.
+ * @returns Sum of prime factors with repetition of p*q.
+ */
+export function wilsonHeight(value: Monzo | FractionValue) {
+  if (Array.isArray(value)) {
+    return dotPrecise(
```

Comment:
Wilson is int valued so we can probably drop support for inaccurate fractional monzos.